firmwares/aeotec/ZWA024-A.json

```
@@ -0,0 +1,31 @@
+{
+	"devices": [
+		{
+			"brand": "aeotec",
+			"model": "ZWA024-A",

```

Should we combine the two files and just call this 
```suggestion
			"model": "ZWA024",
```
?

Seems easier to manage IMO. Region-locking is done by the `"region"` field on each update anyways.